env/env.js

```diff
@@ -2,7 +2,7 @@ const dotenv = require('dotenv');
 const fs = require('fs');
 const path = require('path');
 
-const ENV_FILE = '.production.env';
+const ENV_FILE = '.development.env';
```

Comment:
README says that this change should be applied to make it work locally. Without this change, docker compose was not working properly

Reply:
At runtime, not to be committed. I'm getting the impression that the kludge I made for secrets management works but is non-intuitive. Let's discuss how this could be made better.

Reply:
OK. I think that the code of env/env.js is complicating how env variables are passed. I believe that declaring a .env file is enough to make dotenv load the variables

Reply:
I agree that seemed peculiar to me also and I was not certain of the developer's intentions. It would be better if we could do this in a more standard way.